protos/proto/aptos/transaction/v1/transaction.proto

```diff
@@ -183,9 +183,18 @@ message EventKey {
   string account_address = 2;
 }
 
+// Question[Orderless]: Is this backward compatible?
+// Question[Orderless]: Is it okay to assign number 8 to replay_protection_nonce?
+// Question[Orderless]: TransactionPayload rust struct has ExtraConfig containing multisig_addres, replay_protection_nonce. 
+// We are not including the ExtraConfig directly in protobuf. Instead, we are including multisig_addres as part of multisig payload, 
+// and replay_protection_nonce as part of replay_protector. In case, we add more information in ExtraConfig in future, we might have to change these protobufs and ExtraConfig here.
+// Is that fine? Or would it be confusing?
 message UserTransactionRequest {
   string sender = 1;
-  uint64 sequence_number = 2 [jstype = JS_STRING];
+  oneof replay_protector {
```

Comment:
this is a backward incompatible change

Reply:
actually i'm not that sure about this need to see what the rust changes look like, but it's hard with the crazy amoutn of formatting changes.